server/keyshare/myirmaserver/README

```diff
@@ -0,0 +1,77 @@
+This server provides the primary end points for the myirma web interface. 
```

Comment:
The main README for irmago still needs an update. There, the old keyshare project is still referenced.